### DISCUSSION TITLE
[Feature Request] Gotify support

### DISCUSSION THREAD

**Discussion by kaysond** [jef/streetmerchant#2536] [Ideas and feature requests]
[Gotify](https://gotify.net/) is a self-hosted notification service with a very simple rest API. You'd need two additional env vars: one for the url and another for the token

Just need to do a post to `/message`: https://gotify.net/api-docs

I'll get around to adding this myself eventually if someone else or @jef doesn't get to it first

**Comment by jef** [comment]
Sounds good to me! I use this myself, so it's a good addition. I'll create something later today.

**Reply by jef** [reply]
https://github.com/jef/streetmerchant/issues/2541
https://github.com/jef/streetmerchant/commit/8f2de5dc68667f6b9f0033bed129fd6cdbd9d8a7

Should be good to go! Thanks for the suggestion.

**Reply by kaysond** [reply]
Awesome! Thanks!

I'm gonna submit a quick PR to add an env var for the message priority since it defaults to 0 which then (at least on my phone) doesn't show any notification

**Comment by kaysond** [comment]
Submitted #2546
